Repository not found

Hi, I'm trying to install the tool to try it out and I can't set up the environment.

When running:
```
wit init workspace -a git@github.com:sifive/block-ark.git::wake -a git@github.com:sifive/environment-example-sifive.git
```
I get:
```
Cloning sifive-skeleton...
[ERROR] Command [git clone --no-checkout git@github.com:sifive/sifive-skeleton.git /home/bscuser/BSC/DUH/block-ark/workspace/sifive-skeleton] exited with non-zero exit status [128]
[ERROR] stdout: []
[ERROR] stderr: [Cloning into '/home/bscuser/BSC/DUH/block-ark/workspace/sifive-skeleton'...
ERROR: Repository not found.
fatal: Could not read from remote repository.
```

--- a/wit-manifest.json
+++ b/wit-manifest.json
@@ -1,12 +1,7 @@
 [
     {
-        "commit": "e30f4f12eebc40ac9683eb5a200a2440d8aea87c",
-        "name": "sifive-skeleton",
-        "source": "git@github.com:sifive/sifive-skeleton.git"
-    },
-    {
-        "commit": "152eafed78299de92b8bc45a930b6164c698cc85",
-        "name": "sifive-blocks",
-        "source": "git@github.com:sifive/sifive-blocks.git"
+        "commit": "764559a989e5f6582c365db7f53141c5362a5e9a",
+        "name": "soc-testsocket-sifive",
+        "source": "git@github.com:sifive/soc-testsocket-sifive.git"
     }
 ]

